account_financial_report_qweb/report/general_ledger.py

```diff
@@ -492,6 +497,7 @@ def _inject_account_values(self):
         query_inject_account_params += (
             self.id,
             self.env.uid,
+            self.env.lang,
```

Comment:
That would be redundant given that [`self.context.lang`](https://github.com/odoo/odoo/blob/9.0/openerp/api.py#L802) is itself nothing more than an alias for `self.context.get('lang')`.